TAScheduler/UnitTests/tests.py

```diff
@@ -141,40 +150,239 @@ class TestInstructorLecTAAsmgt(TestCase):
 class TestInstructorLabTAAsmgt(TestCase):
     pass
 
+
 class TestCourseInit(TestCase):
     pass
 
 
 class TestCourseAddInstructor(TestCase):  # Randall
-    pass
+    def setUp(self):
+        self.hold_course = Course.objects.create(
+            course_id=101,
+            semester='Fall 2023',
+            name='Introduction to Testing',
+            description='A course about writing tests in Django.',
+            num_of_sections=3,
+            modality='Online',
+            credits=4
+        )
+        self.tempCourse = CourseObj(self.hold_course)
+        self.instructor_user = User.objects.create(
+            email_address='instructor@example.com',
+            password='password',
+            first_name='Instructor',
+            last_name='User',
+            home_address='123 Instructor St',
+            phone_number='1234567891'
+        )
+        self.instructor_model = Instructor.objects.create(
+            user=self.instructor_user)  # Create an Instructor model instance
+        self.instructor = InstructorObj(self.instructor_model)  # Wrap it with InstructorObj
+
+    def test_add_instructor(self):
+        self.tempCourse.addInstructor(self.instructor)
+        # Check if the instructor was added to the course
+
+        instructor_to_course_exists = InstructorToCourse.objects.filter(
+            instructor=self.instructor_model,  # Use the Instructor model instance
+            course=self.hold_course
+        ).exists()
+        self.assertTrue(instructor_to_course_exists, "Instructor was not added to the course")
 
```

Comment:
Add tests for if instructor has max assignments and if the instructorObj being passed is not actually an instructor object.